### PR TITLE
Rename gRPCEntityOperationError to gRPCStoreError

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by khanhtc1202** [comment]
Sorry, I can not the point of this change, in fact, I think we should keep the current logic "only return gRPCEntityOperationError in case of database error not found" since it mentions the error is an entity not found error, while query on filestore not really get the entity in most of cases, wdyt?

**Comment by knanao** [comment]
@khanhtc1202

> query on filestore not really get the entity in most of cases

I could understand your points but I'm not sure about this one in detail.
IMO,  we don't know what is used as a middleware when we implement around gateway stuff hence I think it's better to wrap that error like this.

**Comment by khanhtc1202** [comment]
> we don't know what a specific store use as a middleware when we implement around gateway stuff

Yes, I agree with that 👍 But my point is instead of extending an existing error that specified "entity" is not found like this to something that returns even in case it is not really an entity (stage log or application live state - based on [this definition](https://pipecd.dev/docs/user-guide/managing-controlplane/architecture-overview/#file-store)), we may define a new one for that. I mean, we should get an entity error in case of requesting for entity, not log. The current `gRPCEntityOperationError` means to errors on actions with data/model object from the database as far as I remember, that why I guess we should not return it on error getting logs 🤔

File: pkg/app/server/grpcapi/web_api.go [hunk 1]
```diff
@@ -1270,8 +1263,8 @@ func (a *WebAPI) DisableStaticAdmin(ctx context.Context, req *webservice.Disable
 	}
 
 	if err := a.projectStore.DisableStaticAdmin(ctx, claims.Role.ProjectId); err != nil {
-		a.logger.Error("failed to disable static admin login", zap.Error(err))
-		return nil, status.Error(codes.Internal, "Failed to disable static admin login")
+		a.logger.Error("failed to disenable static admin login", zap.Error(err))
```
Comment (nghialv) [inline pkg/app/server/grpcapi/web_api.go:1266]:
```suggestion
		a.logger.Error("failed to disable static admin login", zap.Error(err))
```

File: pkg/app/server/grpcapi/web_api.go [hunk 2]
```diff
@@ -1270,8 +1263,8 @@ func (a *WebAPI) DisableStaticAdmin(ctx context.Context, req *webservice.Disable
...
+		return nil, gRPCStoreError(err, "disenable static admin login")
```
Comment (nghialv) [inline pkg/app/server/grpcapi/web_api.go:1267]:
```suggestion
		return nil, gRPCStoreError(err, "disable static admin login")
```

**Review by nghialv — APPROVED** [review]
Thank you.

**Review by khanhtc1202 — APPROVED** [review]
Nice improvement

**Pull request merged**